go/service/chat_local.go

```diff
@@ -251,6 +286,68 @@ func (h *chatLocalHandler) ResolveConversationLocal(ctx context.Context, arg cha
 	}, nil
 }
 
+func (h *chatLocalHandler) resolveConversationsPipeline(ctx context.Context, convs []chat1.Conversation) ([]chat1.ConversationLocal, error) {
+	// Fetch conversation local information in parallel
+	ctx, _ = getUserInfoMapper(ctx, h.G())
+	type jobRes struct {
+		conv  chat1.ConversationLocal
+		index int
+	}
+	type job struct {
+		conv  chat1.Conversation
+		index int
+	}
+	eg, ctx := errgroup.WithContext(ctx)
+	convCh := make(chan job)
```

Comment:
Ah OK. Is it for performance purpose or not leaking number of items in results from server? Because I think goroutines are a bit cheaper than channels